packages/system-tests/src/areas/common.ts

```diff
@@ -223,4 +223,14 @@ export class CommonActions {
   private closeCurrentNotification(): Promise<any> {
     return this.spectron.command('workbench.action.closeMessages');
   }
+
+  private extractTextFromHTML(htmlString: string[]): string[] {
+    const resultArray = [];
+    for (let i = 0; i < htmlString.length; i += 1) {
```

Comment:
Question: how do you feel about using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce? I generally try to avoid for(i=0;...) loops since it's not clear what we are doing. In this case, we are filtering and collecting the result into an array.